vue-app/src/store/index.ts

```diff
@@ -152,11 +153,13 @@ export const mutations = {
       state.recipient = payload.updatedData
     } else {
       state.recipient[payload.step] = payload.updatedData[payload.step]
-      if (payload.stepNumber > state.recipient.furthestStep) {
-        state.recipient.furthestStep = payload.stepNumber
-      }
     }
   },
+  [UPDATE_FORM_PROGRESS](state, nextStep) {
```

Comment:
Same question as above - why the need for another mutation to update `furthestStep`?